dap4/d4shared/build.gradle

```diff
@@ -2,6 +2,9 @@ dependencies {
     compile project(':dap4:d4core')
     compile project(':httpservices')
 
+    compile libraries["jcommander"]
+    compile libraries["slf4j-api"]
+    runtime libraries["slf4j-jdk14"]
```

Comment:
This causes a problem for TDS, as well as for other applications that use the `:dap4:d4shared` module (directly or transitively):
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/Users/cwardgar/dev/projects/thredds2/build/artifacts/tds/exploded/tds-5.0.0-SNAPSHOT.war/WEB-INF/lib/log4j-slf4j-impl-2.4.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/cwardgar/dev/projects/thredds2/build/artifacts/tds/exploded/tds-5.0.0-SNAPSHOT.war/WEB-INF/lib/slf4j-jdk14-1.7.13.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
```
Best practice is to exclude SLF4J bindings in library modules. They should only appear in applications at the "leaves" of the dependency tree, such as TDS and ToolsUI.